src/build.props

```diff
@@ -29,9 +29,9 @@
     place. These properties are actually used by the PowerShell script that
     hides the previous package versions on nuget.org.
     -->
-    <PreviousVersionPrefix>2.0.0-csd.2.beta.2019-04-03</PreviousVersionPrefix>
+    <PreviousVersionPrefix>2.0.0</PreviousVersionPrefix>
```

Comment:
>PreviousVersionPrefix [](start = 5, length = 21)

What is the difference between this and `PreviousPackageVersion` below? Is this still used?

Even if this is still used, it is apparently not a "prefix", since its previous value was `2.0.0-csd.2.beta.2019-04-03`. What is this?

Reply:
oops, this is wrong, thanks for catching this! I forgot to rename it. It should be "PreviousStableSarifVersion". I dont think we have an explicit usage for this  variable today but lets have the right name here!

---
In reply to: [282603779](https://github.com/microsoft/sarif-sdk/pull/1488#discussion_r282603779) [](ancestors = 282603779)

Reply:
reg "PreviousVersionPrefix"  - hmm... we have to interpret it as "PreviousSdkSemanticVersion"

---
In reply to: [282609970](https://github.com/microsoft/sarif-sdk/pull/1488#discussion_r282609970) [](ancestors = 282609970,282603779)

Reply:
The **previous** PR should have changed this to `2.1.0-rtm.0`.

---
In reply to: [282609970](https://github.com/microsoft/sarif-sdk/pull/1488#discussion_r282609970) [](ancestors = 282609970,282603779)